sunkit_dem/base_model.py

```diff
@@ -110,24 +112,22 @@ def kernel(self):
     def kernel(self, kernel):
         if len(kernel) != self.data.cube_like_dimensions[0].value:
             raise ValueError('Number of kernels must be equal to length of wavelength dimension.')
-        if not all([kernel[k].shape == self.temperature_bin_centers.shape for k in kernel]):
+        if not all([k.shape == self.temperature_bin_centers.shape for k in kernel]):
```

Comment:
I was working off this branch using NDCube 2.0.0, and had to revert this change for things to work?

I was getting the error:

```
AttributeError: 'int' object has no attribute 'shape'
```

`kernel` in my case is the AIA response functions [94, 131, 171, 193, 211, 335]

```
print(type(kernel), len(kernel))
<class 'dict'> 6

[k for k in response]
[94, 131, 171, 193, 211, 335]

[kernel[k].shape == temperature_bin_centers.shape for k in kernel]
[True, True, True, True, True, True]
```